app/assets/javascripts/categories.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
There may be differences of industry standard on this matter, but where I work we try to avoid having empty placeholder files.

One way you can avoid files like this ending up in your codebase is to always do 
`git status`
before you commit and then only add the files that you want.

Another great git trick once files are already in the codebase to track only the changes you want is 
`git add -p` 

This allows you to add only bits of file changes if you want to add certain parts but not others. :)